.github/workflows/upload-javadocs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright 2017-2020 O2 Czech Republic, a.s.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+set -e
+
+source $(dirname $0)/functions.sh
+
+if [ "$(is_datadriven_repo ${GITHUB_REPOSITORY})" == "1" ]; then
+   VERSION=$(proxima_version)
+   
+   mvn site -Psite
```

Comment:
Should site (or javadoc) aggregation be called here, or does it happen automatically?

Reply:
Sorry but I don't get the question. `mvn site -Psite` should generate and aggregate javadocs in `target/site/apidocs` which is uploaded in next step. 

Currently I see another issue - that command (`mvn site -Psite`) failed on my ntb, no idea why. Let me try to find out why....